test/test_serialization.py

```diff
@@ -741,6 +741,16 @@ def test_pathlike_serialization(self):
             torch.save(model, path)
             torch.load(path)
 
+    def test_meta_serialization(self):
+        big_model = torch.nn.Conv2d(20000, 320000, kernel_size=3, device='meta')
+
+        with BytesIOContext() as f:
+            torch.save(big_model, f)
+            f.seek(0)
+            state = torch.load(f)
+
+        self.assertEqual(state.weight.size(), big_model.weight.size())
```

Comment:
Do you want to check the device?

Reply:
I think the tensor is plenty big enough :) I would certainly like it if we had more structured serialization tests for different device types, maybe a more involved refactor here is in order.